### ISSUE TITLE
Update 0.6.4 to @supabase/auth-helpers-sveltekit breaks build step due to loglevel.getLogger

### ISSUE THREAD

**Issue by Blade67** [supabase-community/auth-helpers#156]
As mentioned in the title, the recent update `0.6.4` of `@supabase/auth-helpers-sveltekit` breaks the build step. (and potentially more)
Trying to build results in following error:
```sh
[vite-plugin-svelte-kit] loglevel.getLogger is not a function
> loglevel.getLogger is not a function
```
**Downgrading to `0.6.3` resolves this issue.**

**Comment by saturnonearth** [comment]
Thanks, downgrading helped me as well

**Comment by silentworks** [comment]
Fixed in release `0.6.5`.
